### ISSUE TITLE
LocalisationOptions settings are again not working. 

### ISSUE THREAD

**Issue by GrahamDi** [jhomlala/catcher#71]
@kabnfever @jhomlala 
Me again. My dialogues are once again ignoring my localizations and showing the default 'Crash' dialogue. Are you experiencing this? 
I am using:
```
Flutter (Channel stable, v1.12.13+hotfix.5, on Microsoft Windows [Version 10.0.19536.1000], locale en-GB)
```
Tested it with kabnfever's code above and get the default 'Crash' dialogue, not the 'Custom message'.

_Originally posted by @GrahamDi in https://github.com/jhomlala/catcher/issues/53#issuecomment-570780396_

**Comment by GrahamDi** [comment]
I would appreciate your responses on this as my app is about to go live, in 10+ languages and to have the Catcher dialogue only appearing in English is a problem. Thanks.

**Comment by jhomlala** [comment]
Hi, I've just released fix for this in 0.2.9. Please upgrade your Catcher version.
